components/input-validation-mgt/org.wso2.carbon.identity.input.validation.mgt/src/main/java/org/wso2/carbon/identity/input/validation/mgt/model/validators/EmailValidator.java

```diff
@@ -0,0 +1,87 @@
+package org.wso2.carbon.identity.input.validation.mgt.model.validators;
+
+import org.wso2.carbon.identity.input.validation.mgt.exceptions.InputValidationMgtClientException;
+import org.wso2.carbon.identity.input.validation.mgt.model.Property;
+import org.wso2.carbon.identity.input.validation.mgt.model.ValidationContext;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static org.wso2.carbon.identity.input.validation.mgt.utils.Constants.Configs.DEFAULT_EMAIL_REGEX_PATTERN;
+import static org.wso2.carbon.identity.input.validation.mgt.utils.Constants.Configs.IS_EMAIL;
+import static org.wso2.carbon.identity.input.validation.mgt.utils.Constants.ErrorMessages.ERROR_VALIDATION_EMAIL_MISMATCH;
+
+/**
+ * Email format validator.
+ */
+public class EmailValidator extends AbstractRulesValidator {
```

Comment:
```suggestion
public class EmailFormatValidator extends AbstractRulesValidator {
```
Email validator means something validate email not email name. WDYT?

Reply:
Addressed with commit 0b70de75e94c399019b6a3ece166d73f95fcf31d